eng/testing/performance/performance-setup.sh

```diff
@@ -323,7 +334,7 @@ else
     if [[ -n "$perf_fork" ]]; then
         git clone --branch $perf_fork_branch --depth 1 --quiet $perf_fork $performance_directory
     else
-        git clone --branch main --depth 1 --quiet https://github.com/dotnet/performance.git $performance_directory
+        git clone --branch release/7.0 --depth 1 --quiet https://github.com/dotnet/performance.git $performance_directory
```

Comment:
Just as a future thing, do you think it's worth using the build agent variables here?

Reply:
Where do the build agent variables get set/added from? I like the idea, and it may be worth making something in this area that will use the variable if it is properly available. At the very least, it is worthwhile to look into it.

Reply:
If there is a good Doc link to make sure I am thinking of the correct thing when you say build agent variable would be great.

Reply:
See https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml, particularly something like `Build.SourceBranchName` if this doesn't run on PRs, and use `System.PullRequest.TargetBranch` for PRs, that way you don't have to patch when branching.